tests/test_session.py

```diff
@@ -528,7 +528,7 @@ def test_solver_structure(new_solver_session):
     }.issubset(set(dir(solver.fields)))
 
 
-@pytest.mark.fluent_version(">=24.2")
```

Comment:
I think the wider version support is important, can we add the version check within the test code?

Reply:
Sure.

Reply:
Done. Thank you.